src/views/Tools/Tools.js

```diff
@@ -38,10 +39,12 @@ const GET_ALL_TOOLS = gql`
 }
 `
 
-function Tools () {
-  const [searchValue, setSearchValue] = useState('')
-  const [startDate, setStartDate] = useState(new Date('2018-08-02'))
-  const [endDate, setEndDate] = useState(new Date())
+function Tools (props) {
+  const url = new URL(window.location.href)
+
+  const [searchValue, setSearchValue] = useState(url.searchParams.get('searchValue') ? url.searchParams.get('searchValue') : '')
+  const [startDate, setStartDate] = useState(url.searchParams.get('startDate') ? new Date(url.searchParams.get('startDate')) : new Date('2018-08-02'))
+  const [endDate, setEndDate] = useState(url.searchParams.get('endDate') ? new Date(url.searchParams.get('endDate')) : new Date())
```

Comment:
Why are you hard-coding `2018-08-02` for `startDate` and not hard-coding an endDate? Why are you hard-coding dates at all?

Also, inside `useState`, it's better to use [JS short-circuit evaluation](https://codeburst.io/javascript-what-is-short-circuit-evaluation-ff22b2f5608c) to assign values. For instance, `useState(new Date(url.searchParams.get('startDate')) || new Date('2018-08-02'))`.